decidim-debates/app/views/decidim/debates/debates/index.html.erb

```diff
@@ -8,7 +8,7 @@
     <% end %>
   <% end %>
 
-  <%= render partial: "decidim/shared/filters", locals: { filter_sections:, search_variable: } %>
+  <%= render partial: "decidim/shared/filters", locals: { filter_sections:, search_variable:, skip_to_id: "debates" } %>
```

Comment:
There is no input, and the sorting order is not being preserved.

Reply:
Fixed